src/panel/direction/DirectionInput.jsx

```diff
@@ -80,6 +81,7 @@ class DirectionInput extends React.Component {
   render() {
     const { pointType, inputRef, isLoading } = this.props;
     const { mounted, readOnly } = this.state;
+    const isMobile = isMobileDevice();
```

Comment:
Same component as before. For this to be reactive to screen size changes, you have to use the context, not the function directly. An example, on another class component https://github.com/QwantResearch/erdapfel/blob/master/src/panel/poi/PoiPanel.jsx#L332